===== Patch instructions generated by SAVER =====
[+] { Insert: if (true) free(*(do_rsync:s)) at 4 (line 64, column 2) }
===== Actual patch in diff format =====
--- a/table1/benchmarks/lxc/src/lxc/bdev/lxcrsync.c
+++ b/table1/benchmarks/lxc/src/lxc/bdev/lxcrsync.c
@@ -62,6 +62,7 @@ int do_rsync(const char *src, const char *dest)
 	s[l-1] = '\0';
 
 	execlp("rsync", "rsync", "-aHXS", "--delete", s, dest, (char *)NULL);
+  free(s);
 	exit(1);
 }
 
